website/tutorial/managing-your-application-resources.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 6
+title: Managing your application resources
+description: Cover useful actions available to manage your application resources
+keywords: [podman desktop, podman, pod, Containerfile, Dockerfile]
+tags: [podman-desktop, managing-resources, managing-kubernetes-resources]
+---
+
+# Managing your application resources
+
+This tutorial covers the following tasks that you can perform to manage your application resources visually:
+
+- Managing containers and pods
+- Managing images
+- Managing volumes
+- Managing Kubernetes objects
+- Managing other resources
+
+Several useful actions are available in the UI to make resource management easy.
+
+## Before you begin
+
```

Comment:
Maybe we need a `Make sure you have:` 

because now its reads as: "Before you begin Installed Podman Desktop"

Reply:
Thanks for this suggestion.